SpecialDiscount.cs

```diff
@@ -1,14 +1,16 @@
 ﻿namespace Price_Calculator_Kata
 {
-    public class SpecialDiscountPair
+    public class SpecialDiscount
     {
         public int UPC { get; set; }
         public double Percentage { get; set; }
+        public Type DiscountType { get; set; }

```

Comment:
Since the enum should be renamed to `DiscountType`, the field should be named with different name (I'm not sure about it), but I prefer to use `public DiscountType Type` as the `DiscountType` is an enum that could exist by itself, so you need to be explicit that it's a `DiscountType`, but the `Type` is used under `SpecialDiscount` and `UniversalDiscount` so it would be accessed like this: `SpecialDiscount.Type` and `UniversalDiscount.Type`